tests/pages/build-list.js

```diff
@@ -17,6 +17,7 @@ export default create({
 
   builds: collection('.build-list .pr-row', {
     name: text('.build-info a'),
+    badge: text('.badge'),
```

Comment:
We are aiming to stick with [test selectors](https://ember-test-selectors.com/) and avoid using CSS classes for testing purposes. Just add an attribute to the element, something like `data-test-draft-badge` and then select it in test with attribute selector: `[data-test-draft-badge]`. This helps to make tests independent from styling. Also the add-on removes all test selectors from production builds.

Reply:
Cool, didn't know that